src/main/java/fr/mossroy/tomcat/tools/ExternalPropertySource.java

```diff
@@ -31,8 +33,9 @@
     private static final String CATALINA_PROPERTIES = "conf/catalina.properties";
     private static final String CATALINA_PROPERTIES_FILE_PROPERTY = "fr.mossroy.tomcat.tools.ExternalPropertySource.file";
     private static final Log LOGGER = LogFactory.getLog(ExternalPropertySource.class);
+    private static final Pattern regExp = Pattern.compile("\\$\\{([^\\}]*)\\}");

```

Comment:
`regExp` is not a very meaningful variable name : could you use one that gives more hints on what it does?
It also does not follow the naming convention of a constant in Java
For example `REGEXP_PLACEHOLDER_DOLLAR_CURLY_BRACES`